pkg/kubelet/cm/deviceplugin/device_plugin_handler.go

```diff
@@ -337,19 +343,17 @@ func (h *HandlerImpl) writeCheckpoint() error {
 	if err != nil {
 		return err
 	}
-	filepath := h.devicePluginManager.CheckpointFile()
-	return ioutil.WriteFile(filepath, dataJSON, 0644)
+	return h.store.Write(defaultCheckpoint, dataJSON)
 }
 
 // Reads device to container allocation information from disk, and populates
 // h.allocatedDevices accordingly.
 func (h *HandlerImpl) readCheckpoint() error {
-	filepath := h.devicePluginManager.CheckpointFile()
-	content, err := ioutil.ReadFile(filepath)
+	content, err := h.store.Read(defaultCheckpoint)
 	if err != nil && !os.IsNotExist(err) {
-		return fmt.Errorf("failed to read checkpoint file %q: %v", filepath, err)
+		return fmt.Errorf("failed to read checkpoint file %q: %v", defaultCheckpoint, err)
 	}
-	glog.V(2).Infof("Read checkpoint file %s\n", filepath)
+	glog.V(2).Infof("Read checkpoint file %s\n", defaultCheckpoint)
```

Comment:
Not sure if there is a convention about the verbose level to use.
My feeling is V(2) can be changed to at least V(4).